tests/ubuntu_test.yaml

```diff
@@ -0,0 +1,21 @@
+schemaVersion: '1.0.0'
+commandTests:
+- name: 'apt-get'
+  command: 'apt-get'
+  args: ['help']
+  excludedError: ['.*FAIL.*']
+  expectedOutput: ['.*Usage.*']
+- name: 'apt-config'
+  command: 'apt-config'
+  args: ['dump']
+  expectedOutput: ['Acquire::Retries "3"']
+  name: 'apt-config'
+- name: 'path'
+  command: 'sh'
+  args: ['-c', 'echo $PATH']
+  expectedOutput: ['/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin']
+fileExistenceTests:
+- name: 'Root'
+  isDirectory: true
```

Comment:
this field also is deprecated in the latest schema version, in favor of using a unix-style permissions string (optional)

Reply:
I think this has to be updated in the debian{#}_test as well

Reply:
The new bazel rule should support both 1 and 2. But yeah, we should move everything to 2.